thoth/storages/graph/postgres.py

```diff
@@ -6517,7 +6517,7 @@ def sync_provenance_checker_result(self, document: dict) -> None:
                 # Discover unresolved package matching the message ID
                 if package["id"] == "MISSING-PACKAGE":
 
-                    python_package_version_entity = self._create_python_package_version(
+                    self._create_python_package_version(
```

Comment:
I agree on keeping the unused variable `provenance_checker_run` in this method as this looks like an optimization to persist the data within the session scope :+1: This has been used in other methods where `get_or_create` was called as well in this class.

Reply:
In fact, I did not change that line (provenance_checker_run) because it was not flagged by flake8. It's flagged by pylint though apparently, not sure why there is a discrepancy...